webpack.config.js

```diff
@@ -0,0 +1,54 @@
+const webpack = require("webpack");
```

Comment:
should be based on the latest standard like
https://github.com/FlockOfBirds/mobile-device/blob/master/webpack.config.js